lib/workers/repository/process/extract-update.ts

```diff
@@ -141,7 +142,10 @@ export async function extract(
   const cachedExtract = cache.scan[baseBranch!];
   const configHash = fingerprint(generateFingerprintConfig(config));
   // istanbul ignore if
```

Comment:
doesn't work, can you do these refactorings in seperate PR and add tests?

Reply:
https://github.com/renovatebot/renovate/pull/35941